starport/pkg/events/events.go

```diff
@@ -2,41 +2,66 @@
 // for others to consume and display to end users in meaningful ways.
 package events
 
-import "fmt"
+import (
+	"fmt"
+
+	"github.com/gookit/color"
+)
 
 // Event represents a state.
-type Event struct {
-	// Status shows the current status of event.
-	status Status
+type (
+	Event struct {
+		// Description of the state.
+		Description string
 
-	// Description of the state.
-	Description string
-}
+		// Status shows the current status of event.
+		Status Status
 
-// Status shows if state is ongoing or completed.
-type Status int
+		// Color of the text.
+		Color color.Color
+	}
+
+	// Status shows if state is ongoing or completed.
+	Status int
+
+	// Option event options
+	Option func(*Event)
+)
 
 const (
 	StatusOngoing Status = iota
 	StatusDone
+	StatusInfo
 )
 
+// Color sets the text color
+func Color(c color.Color) Option {
```

Comment:
Hm instead of colors can we do `Icon` and make it accept one of OK, NotOK, Bullet, Info?